service/component-samples/owner/src/main/java/org/fido/iot/sample/OwnerContextListener.java

```diff
@@ -114,7 +114,7 @@ protected void dispatching(Composite request) {
       protected void failed(Exception e) {
         StringWriter writer = new StringWriter();
         try (PrintWriter pw = new PrintWriter(writer)) {
-          e.printStackTrace(pw);
+          System.out.println("Failed to write data: " + e.getMessage());
```

Comment:
Rather than use System.out to print log messages, why not use the logger?  That way everything will go to the same place.

Reply:
Updated.